ports/stm32/boards/STM32L496GDISC/mpconfigboard.h

```diff
@@ -0,0 +1,51 @@
+#define MICROPY_HW_BOARD_NAME       "L496G-DISCO"
+#define MICROPY_HW_MCU_NAME         "STM32L496"
+
+#define MICROPY_HW_HAS_SWITCH       (1)
+#define MICROPY_HW_ENABLE_RNG       (1)
+#define MICROPY_HW_ENABLE_RTC       (1)
+#define MICROPY_HW_ENABLE_TIMER     (1)
+#define MICROPY_HW_ENABLE_USB       (1)
+
+// MSI is used and is 4MHz
```

Comment:
Can you please add to this comment what the resulting MCU freq is.

Reply:
I will introduce this info in the final commit. The MCU core frequency will be (as on any l4) 80MHz.